Add missing dependency in Makefile
--- a/Makefiles/shared/shared.mk
+++ b/Makefiles/shared/shared.mk
@@ -47,8 +47,9 @@
 
 
 clean_shared: clean_templates clean_man1 clean_setup_scripts
+	$(RM) -f setup/templates/header-*.sh.in
+	$(RM) -f setup/templates/header-*.sh
 build_shared: build_templates build_man1 build_setup_scripts
-	rm -f setup/templates/header.sh
 
 install_shared: $(patsubst %, install_%,$(SHARED_ACTIONS)) install_setup_scripts
 setup_shared: run_setup_scripts
@@ -79,10 +80,12 @@
 				     $(CGIDIR_FILES) \
 				     $(WWWDIR_FILES) \
 				     $(MODULE_SETUP_SOURCE_FILES) \
-				     setup/templates/header.sh.in \
+				     setup/templates/header-$(MODULE).sh.in \
 			)
 TEMPLATE_BUILDED_FILES=$(patsubst %.in,%,$(TEMPLATE_SOURCE_FILES))
 
+setup/templates/header-$(MODULE).sh.in: setup/templates/header.sh.in
+	cp $^ $@
 
 build_templates: $(TEMPLATE_BUILDED_FILES)
 
@@ -140,8 +143,8 @@
 
 build_setup_scripts: $(MODULE_SETUP_BUILDED_FILES)
 
-$(MODULE_SETUP_BUILDED_FILES): $(MODULE_SETUP_TOBUILD_FILES)
-	cat setup/templates/header.sh $< > $@
+$(MODULE_SETUP_BUILDED_FILES): $(MODULE_SETUP_TOBUILD_FILES) setup/templates/header-$(MODULE).sh
+	cat setup/templates/header-$(MODULE).sh $< > $@
 
 clean_setup_scripts:
 	-rm -f $(MODULE_SETUP_BUILDED_FILES)
