Index: oar/sources/core/tools/oardo.c
===================================================================
--- oar.orig/sources/core/tools/oardo.c	2013-09-19 23:06:57.461518535 +0200
+++ oar/sources/core/tools/oardo.c	2013-09-19 23:07:28.034101201 +0200
@@ -123,5 +123,6 @@
     // execute the OAR command with oar privileges
     execv(CMD_WRAPPER, av);
     error("Cannot run with oar privileges "CMD_WRAPPER, 1);
+    return 0;
 }
 
