Bash snipsets to be sourced are not binaries
diff --git a/Makefiles/shared/shared.mk b/Makefiles/shared/shared.mk
index 7a484262..2fbb0755 100644
--- a/Makefiles/shared/shared.mk
+++ b/Makefiles/shared/shared.mk
@@ -151,7 +151,7 @@ uninstall_setup_scripts:
 
 $(MODULE_SETUP_TARGET_FILES): $(MODULE_SETUP_BUILDED_FILES)
 	install -d $(DESTDIR)$(OARDIR)/setup
-	install -m 0755 $< $@
+	install -m 0644 $< $@
 
 #
 # OAR_PERLLIB
diff --git a/setup/templates/header.sh.in b/setup/templates/header.sh.in
index 9f08dec3..584b0e7e 100644
--- a/setup/templates/header.sh.in
+++ b/setup/templates/header.sh.in
@@ -1,4 +1,3 @@
-#! /bin/sh
 
 set -e
 
