Index: oar/setup/node.sh.in
===================================================================
--- oar.orig/setup/node.sh.in	2013-09-19 23:32:27.000000000 +0200
+++ oar/setup/node.sh.in	2013-09-19 23:34:32.449060061 +0200
@@ -47,11 +47,13 @@
 node_setup() {
     install_conffile \
         ${SHAREDIR}/epilogue \
-        ${OARCONFDIR}/epilogue
+        ${OARCONFDIR}/epilogue \
+        0755
     
     install_conffile \
         ${SHAREDIR}/prologue \
-        ${OARCONFDIR}/prologue
+        ${OARCONFDIR}/prologue \
+        0755
 
     install_conffile \
         ${SHAREDIR}/sshd_config \
Index: oar/setup/server.sh.in
===================================================================
--- oar.orig/setup/server.sh.in	2013-09-19 23:32:27.000000000 +0200
+++ oar/setup/server.sh.in	2013-09-19 23:34:32.449060061 +0200
@@ -52,11 +52,13 @@
 
     install_conffile \
         ${SHAREDIR}/server_prologue \
-        ${OARCONFDIR}/server_prologue
+        ${OARCONFDIR}/server_prologue \
+        0755
 
     install_conffile \
         ${SHAREDIR}/server_epilogue \
-        ${OARCONFDIR}/server_epilogue
+        ${OARCONFDIR}/server_epilogue \
+        0755
 
     setup_oar_ssh
     create_phoenix_home
